.github/workflows/file-live-demo-issues.yml

```diff
@@ -0,0 +1,68 @@
+name: Manually trigger issues for using in live demonstrations
+
+on:
+  workflow_dispatch:
+    inputs:
+      demo_lead_username:
+        description: The github username of who will be performing the live demonstrations
+        required: true
+      demo_support_username:
+        description: The github username of who will file the PR to support the live code demonstration
+        required: true
```

Comment:
Assuming that material will get split up the same way strikes me as too strong of an assumption.